src/everest/bin/everest_script.py

```diff
@@ -133,7 +133,9 @@ async def run_everest(options):
             options.config.dump(save_config_path)
         except (OSError, LookupError) as e:
             logging.getLogger(EVEREST).error(f"Failed to save optimization config: {e}")
-        await start_server(options.config, options.debug)
+
+        logging_level = logging.DEBUG if options.debug else options.config.logging_level
+        await start_server(options.config, logging_level)
```

Comment:
Perhaps add a new test for this new way of specifying the logging in `test_everest_entry.py `?